cpp/IceStorm/replicated2/Publisher.cpp

```diff
@@ -1,10 +1,6 @@
 // Copyright (c) ZeroC, Inc.
 
-#ifdef _MSC_VER
-// For localtime warning
-#    define _CRT_SECURE_NO_WARNINGS
-#endif
-
+#include "../../common/time.h"
```

Comment:
I would prefer a simpler `#include "Time.h"`

Reply:
I rater use "../../common". It's where the file is and I'd rather not add an include directory to each target, makes the cmake file uglier.